.ci/travis.sh

```diff
@@ -65,7 +65,7 @@ checkstyle-regression)
   mvn -e test -Dtest=CheckstyleRegressionTest#setupFiles -Dregression-path=../
   cd ../
   cd checkstyle
-  mvn -e clean verify -e -DskipTests -DskipITs -Dpmd.skip=true -Dfindbugs.skip=true -Dcobertura.skip=true
+  mvn -e clean verify -e -DskipTests -DskipITs -Dpmd.skip=true -Dfindbugs.skip=true -Dcobertura.skip=true -Dmaven.sevntu-checkstyle-check.checkstyle.version=8.7
```

Comment:
This is needed because checkstyle is downgrading sevntu's run before API breaking compatibility.
Once checkstyle upgrades this can be removed.